__tests__/app.test.js

```diff
@@ -142,6 +142,22 @@ describe("GET - /api/articles", () => {
             expect(body.articles).toBeSortedBy('comment_count', { descending: true })
         })
     })
+    it("GET:200 - responds with articles sorted by topics in default order", () => {
+        return request(app)
+        .get("/api/articles?sort_by=topic")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toBeSortedBy('topic', { descending: true })
+        })
+    })
+    it("GET: - responds with articles sorted by topics in ascending order", () => {
+        return request(app)
+        .get("/api/articles?sort_by=topic&order=asc")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toBeSortedBy('topic', { ascending: true })
+        })
+    })
```

Comment:
Your logic in your model is good and filters by topic, but I can't see any tests that check if the returned articles have been successfully filtered?